pandas/core/frame.py

```diff
@@ -105,7 +105,8 @@
     axes_single_arg="{0 or 'index', 1 or 'columns'}",
     optional_by="""
         by : str or list of str
-            Name or list of names which refer to the axis items.""")
+            Name or list of names which refer to the axis items.""",
```

Comment:
I think this is fine (given our current way of doing this with `_shared_doc_kwargs`. This could maybe get an overhaul, but that is another issue :-))